packages/platforms/accton/x86-64/as5915-18x/onlp/builds/x86_64_accton_as5915_18x/module/inc/x86_64_accton_as5915_18x/x86_64_accton_as5915_18x_config.h

```diff
@@ -0,0 +1,137 @@
+/**************************************************************************//**
```

Comment:
Are the // needed in the end?

Reply:
It looks like this is one of the comment style of doxygen.
https://www.doxygen.nl/manual/docblocks.html#cppblock